main.py

```diff
@@ -536,7 +536,41 @@ def populate_memes():
                     tmp, FEEDBACK_DEL_TIMER, error=True, call_msg=message
                 )
 
-
+    elif message.content.startswith("!patreon"):
+        is_donator = False
+        for r in message.author.roles: #check if donator
+            if r.name.lower() == "donor":
+                is_donator = True
+        if is_donator:
+            colour_hex = message.content[10:]
+            if not len(colour_hex) or not message.content[8:10] == " #":
+                tmp = await client.send_message(message.channel, "Usage: **!patreon #ff0000** is red\n**!patreon #00ff00** is green\netc...")
```

Comment:
I think ` is better in such a message than **

Reply:
Ok but feel free to change the error messages later to whatever, I don't like them too.